integration/tests/accessibility.test.ts

```diff
@@ -20,17 +20,17 @@
 import { common } from '../page_objects/common';
 
 describe('Accessibility tree', () => {
-  it('should include the series types if one type of series', async () => {
-    const tree = await common.testAccessibilityTree(
-      'http://localhost:9001/iframe.html?id=annotations-lines--x-continuous-domain',
-      '.echCanvasRenderer',
-    );
-    // the legend has bars and lines as value.descriptions not value.name
-    const hasTextOfChartTypes = tree.children.filter((value) => {
-      return value.name === 'bar chart';
-    });
-    expect(hasTextOfChartTypes[0].name).toBe('bar chart');
-  });
+  // it('should include the series types if one type of series', async () => {
+  //   const tree = await common.testAccessibilityTree(
+  //     'http://localhost:9001/iframe.html?id=annotations-lines--x-continuous-domain',
+  //     '.echCanvasRenderer',
+  //   );
+  //   // the legend has bars and lines as value.descriptions not value.name
+  //   const hasTextOfChartTypes = tree.children.filter((value) => {
+  //     return value.name === 'bar chart';
+  //   });
+  //   expect(hasTextOfChartTypes[0].name).toBe('bar chart');
+  // });
```

Comment:
`xit` (aka `it.skip` or `xtest`) is much cleaner way to skip tests, less code line diff. One exception is if the test has a type error. See https://jestjs.io/docs/api#testskipname-fn

```suggestion
  xit('should include the series types if one type of series', async () => {
    const tree = await common.testAccessibilityTree(
      'http://localhost:9001/iframe.html?id=annotations-lines--x-continuous-domain',
      '.echCanvasRenderer',
    );
    // the legend has bars and lines as value.descriptions not value.name
    const hasTextOfChartTypes = tree.children.filter((value) => {
      return value.name === 'bar chart';
    });
    expect(hasTextOfChartTypes[0].name).toBe('bar chart');
  });
```

Reply:
Thank you!! The linter changed xit -> it.skip 22bfe35